docs/docs/guides/installing-the-cli.md

```diff
@@ -85,23 +85,11 @@ The Kurtosis CLI cannot be installed directly on Windows. Windows users are enco
 
 </Tabs>
 
-III. Add tab completion
+III. Add command-line completion
```

Comment:
```suggestion
III. [Optional] Add command-line completion
```

Just a suggestion. I can be convinced otherwise but this step is a quality-of-life thing and I want to be honest to the user about it

Reply:
I'd prefer not to highlight it in the header so that people don't _immediately_ skip it (big red button thing again), but I'd put it in the text of the step:
```
III. Add command-line completion
--------------------------------
Kurtosis supports command-line completion to allow completing subcommands and dynamic values (e.g. enclave name during `enclave inspect`). This isn't required, but we believe it significantly enhances the Kurtosis experience. If you'd like to install it, see [these instructions][installing-command-line-completion].
```